index.js

```diff
@@ -42,24 +42,15 @@ module.exports = {
     typescript: require('./lib/config/typescript'),
 
     // Augmenting configs - When extending, these go after the core config
-    ava: require('./lib/config/ava'),
-    'eslint-comments': require('./lib/config/eslint-comments'),
-    flow: require('./lib/config/flow'),
     graphql: require('./lib/config/graphql'),
     jest: require('./lib/config/jest'),
     jquery: require('./lib/config/jquery'),
     lodash: require('./lib/config/lodash'),
-    mocha: require('./lib/config/mocha'),
```

Comment:
What do you think, should we just go whole-hog and remove more of the ones we basically don't use anymore, like `lodash`/ `jquery`?

Reply:
I do.